languages/i18n/vp.json

```
@@ -2658,12 +2658,12 @@
 	"sp-contributions-concurrency-user": "Sorry, too many requests are being made from your user account. Please try again later.",
 	"sp-contributions-concurrency-ip": "Sorry, too many requests are being made from your IP address. Please try again later.",
 	"whatlinkshere": "Ka zedvera her made",
-	"whatlinkshere-count": "Displayed $1 {{PLURAL:$1|item|items}}.",
+	"whatlinkshere-count": "Mahsedan $1 {{PLURAL:$1|item|items}}.",

```

```suggestion
	"whatlinkshere-count": "Mahsedan $1 vaska.",
```
(Gruun li nai, afto fras ti sejena na « Mahsedan 5 items »)